src/MasterSite.php

```diff
@@ -118,6 +118,7 @@ protected function hooks() {
 		add_action( 'admin_menu', [ $this, 'add_restricted_tags_box' ] );
 		add_action( 'do_meta_boxes', [ $this, 'remove_default_tags_box' ] );
 		add_action( 'pre_insert_term', [ $this, 'disallow_insert_term' ], 1, 2 );
+		add_action( 'wp_dropdown_users_args', [ $this, 'filter_authors' ] );
```

Comment:
The `wp_dropdown_users_args` is a [filter](https://developer.wordpress.org/reference/hooks/wp_dropdown_users_args/), so it will be use with `apply_filters ()` function.